components/About/About.tsx

```diff
@@ -5,6 +5,24 @@ import { Content } from "../../layout/Content";
 
 import { SingleCard } from "./SingleCard";
 
+const CARDS_TITLES = {
```

Comment:
why don't create array of objects wuth title and description, keep it in seperate file (i like to have a data folder if not external solution like CMS/DB is used — but its not that important in this case I guess) and then use a map method to render all of the items that looks the same but have just different texts?

Reply:
Done